tools/Evm/Evm/T8n/T8nInputReader.cs

```diff
@@ -76,7 +76,11 @@ private static InputData ReadStdInput()
         using var reader = new StreamReader(Console.OpenStandardInput());
         try
         {
-            return EthereumJsonSerializer.Deserialize<InputData>(reader.ReadToEnd());
+            var jsonText = reader.ReadToEnd();
+            TransactionMetaDataWrapper txsMetaDataWrapper = EthereumJsonSerializer.Deserialize<TransactionMetaDataWrapper>(jsonText);
```

Comment:
Do we have to `ReadToEnd`? Can we deserialize from `Stream` or `Reader` as we read it?